examples/runtime/starlark/inline/fn-config.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: fn.kpt.dev/v1beta1
+kind: StarlarkFunction
+metadata:
+  name: my-star-fn
+  annotations:
+    config.k8s.io/function: |
+      container:
+        image: gcr.io/kpt-fn/starlark:unstable
+source: |
+  # set the namespace on each resource
+  def run(r, ns_value):
```

Comment:
do you need a function definition? Can you inline this?

Reply:
Echo from https://github.com/bazelbuild/starlark/blob/master/spec.md#for-loops:
> In Starlark, a for loop is permitted only within a function definition. A for loop at top level results in a static error.